server/src/test/java/org/apache/iotdb/db/engine/cache/BloomFilterCacheTest.java

```diff
@@ -0,0 +1,195 @@
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.iotdb.db.engine.cache;
+
+import org.apache.iotdb.tsfile.file.metadata.enums.TSDataType;
+import org.apache.iotdb.tsfile.file.metadata.enums.TSEncoding;
+import org.apache.iotdb.tsfile.fileSystem.FSFactoryProducer;
+import org.apache.iotdb.tsfile.read.common.Path;
+import org.apache.iotdb.tsfile.utils.BloomFilter;
+import org.apache.iotdb.tsfile.write.TsFileWriter;
+import org.apache.iotdb.tsfile.write.record.Tablet;
+import org.apache.iotdb.tsfile.write.schema.IMeasurementSchema;
+import org.apache.iotdb.tsfile.write.schema.Schema;
+import org.apache.iotdb.tsfile.write.schema.UnaryMeasurementSchema;
+
+import org.apache.commons.io.FileUtils;
+import org.junit.*;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BloomFilterCacheTest {
+  private List<String> pathList;
+  private int pathSize = 3;
+  private BloomFilterCache bloomFilterCache;
+
+  @Before
+  public void setUp() throws Exception {
+    // set up and create tsFile
+    pathList = new ArrayList<>();
+    for (int i = 0; i < pathSize; i++) {
+      String path =
+          "target"
+              .concat(File.separator)
+              .concat("data")
+              .concat(File.separator)
+              .concat("data")
+              .concat(File.separator)
+              .concat("sequence")
+              .concat(File.separator)
+              .concat("root.sg" + (i + 1))
+              .concat(File.separator)
+              .concat("0")
+              .concat(File.separator)
+              .concat("0")
+              .concat(File.separator)
+              .concat("1-0-0-0.tsfile");
+      String device = "d" + (i + 1);
+      pathList.add(path);
+      createTsFile(path, device);
+    }
+    bloomFilterCache = BloomFilterCache.getInstance();
+  }
+
+  @After
+  public void tearDown() {
+    try {
+      for (String filePath : pathList) {
+        FileUtils.forceDelete(new File(filePath));
+      }
+    } catch (IOException e) {
+      Assert.fail(e.getMessage());
+    }
+  }
+
+  @Test
+  public void testGet() {
+    try {
+      for (String filePath : pathList) {
+        BloomFilter bloomFilter =
+            bloomFilterCache.get(new BloomFilterCache.BloomFilterCacheKey(filePath));
+        Assert.assertNotEquals(bloomFilter, null);
```

Comment:
I think it's better to use `FileReaderManager.getInstance().get()` to get a `TsFileSequenceReader`, and use that to read a `BloomFilter`, then compare the content of these two bloomfilters. Otherwise, we only know it is not null, but we don't know whether the content of it is correct.